.konflux/cdsr-webhook/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:1.20.12 AS builder
+WORKDIR /go/src/github.com/openshift/csi-driver-shared-resource
+# to make SAST/SNYK happy
+RUN rm -rf examples
+RUN rm -f vendor/k8s.io/apimachinery/pkg/util/managedfields/pod.yaml
+COPY . .
+RUN rm -rf /go/src/github.com/openshift/csi-driver-shared-resource/examples
+RUN rm -f /go/src/github.com/openshift/csi-driver-shared-resource/vendor/k8s.io/apimachinery/pkg/util/managedfields/pod.yaml
```

Comment:
I would rather us use a `.dockerignore` file at the project root - but we can address in a follow-up pull request.

https://docs.docker.com/build/building/context/#dockerignore-files